extract_setup_data.py

```diff
@@ -0,0 +1,81 @@
+import yaml
+import random
+
+
+# This will only work if the setup for your yaml file is structured the same way as the
+# given yaml files.
+
+def load_yaml(player_count):
+    """Loads the setup for the passed player count into Python
+
+    Parameters
+    ----------
+    player_count: int
+        The number of players in the game
+
+    Returns
+    -------
+    setup
+        An dict representing the loaded yaml file
+    """
+
+    file_name = ('etc/' + str(player_count) + '.yaml')
```

Comment:
this can be rewritted as `f"etc/{player_count}.yaml"` its both faster, more readable, and much more pythonic

Reply:
(called an fstring)